assets/javascripts/modules/autoComplete.js

```diff
@@ -314,28 +323,99 @@ var suggestionsEvent = function () {
   });
 };
 
+
+/**
+  * Generate random variable name to ensure no conflicts
+*/
+var randomVariableName = function () {
```

Comment:
we don't need this `randomVariableName` functionality right? just place it on `window.suggestions` and at the moment we don't support 2x autoCompletes on the same page. As discussed the placing this on `window` will be removed in future work